geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/connection/AuthIntegrationTest.java

```diff
@@ -117,15 +118,19 @@ private void setupCache(String username, boolean withSecurityManager) throws Exc
     cf.set(LOG_LEVEL, "error");
     cf.set(MCAST_PORT, "0");
     cf.set(LOCATORS, "");
-    if (username != null) {
-      cf.set(ConfigurationProperties.GEODE_FOR_REDIS_USERNAME, username);
-    }
     if (withSecurityManager) {
       cf.set(ConfigurationProperties.SECURITY_MANAGER, SimpleSecurityManager.class.getName());
     }
     cache = cf.create();
     port = AvailablePortHelper.getRandomAvailableTCPPort();
-    server = new GeodeRedisServer("localhost", port, (InternalCache) cache);
+
+    TestRedisConfiguration.Builder redisConfigBuilder = TestRedisConfiguration.builder();
+    redisConfigBuilder.withAddress("localhost").withPort(port).build();
```

Comment:
I think the `build()` at the end of this is not necessary here.

Reply:
Yes, you're correct.